src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/UpdateConsentedCaseController.java

```diff
@@ -82,8 +82,7 @@ public ResponseEntity<AboutToStartOrSubmitCallbackResponse> updateDetails(
         log.info("DEBUGGING NOC - updateApplicantOrSolicitorContactDetails entered and applicant name is still present",
             caseDetails.getData().get(APPLICANT_FIRST_MIDDLE_NAME) != null);
 
-        if (featureToggleService.isCaseworkerNoCEnabled()
-            && ofNullable(caseDetails.getData().get(INCLUDES_REPRESENTATION_CHANGE)).isPresent()
+        if (ofNullable(caseDetails.getData().get(INCLUDES_REPRESENTATION_CHANGE)).isPresent()
```

Comment:
This can be simplified to
`if (YES_VALUE.equals(caseDetails.getData().get(INCLUDES_REPRESENTATION_CHANGE))) {`

String.equals will return false if the argument is null

Reply:
Legacy code but agreed, I'll do the refactor.